components/style/rule_tree/mod.rs

```diff
@@ -580,13 +581,13 @@ impl StrongRuleNode {
                 let existing =
                     next_sibling_ptr.compare_and_swap(ptr::null_mut(),
                                                       new_ptr,
-                                                      Ordering::SeqCst);
+                                                      Ordering::Relaxed);
```

Comment:
I'm not completely sure how that could happen, tbh. I can't imagine a reordering that would make sense there, given it's across a loop. Anyway, using `Aquire` seems safer, so let's just use that.